samples/account-linking/csharp/.tours/messaging-extension-account-linking.tour

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Messaging Extension Account Linking",
+  "steps": [
+    {
+      "file": "Source/Bots/SampleActivityHandler.cs",
+      "description": "Account linking with messaging extensions is a bit easier than tabs. For this sample, we demonstrate how to do this with a [messaging extension search command](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/search-commands/define-search-command). \r\n\r\nThe flow will be nearly identical with all messaging extensions so this should be adaptable for [action commands](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/action-commands/define-action-command?tabs=AS) or [link unfurling](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/link-unfurling?tabs=dotnet).",
+      "line": 65
+    },
+    {
+      "file": "Source/Bots/SampleActivityHandler.cs",
+      "description": "As before we get the user's id (Azure Ad id, not Bot Framework id) and their tenant id. Unlike in the tab flow, these two values are trustworthy in the inbound payload so we don't need an Azure AD access token to authenticate the user.\r\n\r\n",
```

Comment:
> "Unlike in the tab flow, these two values are trustworthy"
Might be good to explain why these are trust worthy. Even something like: "Unlike in the tab flow, these two values come from Bot Framework so are trustyworthy..."